src/SFML/Graphics/Font.cpp

```diff
@@ -129,7 +129,7 @@ m_fontHandles(copy.m_fontHandles),
 m_isSmooth(copy.m_isSmooth),
 m_info(copy.m_info),
 m_pages(copy.m_pages),
-m_pixelBuffer(copy.m_pixelBuffer)
+m_pixelBuffer(copy.m_pixelBuffer.begin(), copy.m_pixelBuffer.end())
```

Comment:
Why change this from how it was before?

Reply:
Ah, I didn't compare it to master; I'll change it to conform.

Reply:
What would be incorrect about defaulting this special member function?